utils/correction.py

```diff
@@ -0,0 +1,76 @@
+import gc
+import numpy as np
+import awkward as ak
+import gzip
+import pickle
+
+from coffea.lookup_tools.lookup_base import lookup_base
+
+from coffea.lookup_tools import extractor
+
+
+from coffea.lumi_tools import LumiMask
+from coffea.btag_tools import BTagScaleFactor
+from helpers.cTagSFReader import *
+def build_lumimask(filename):
```

Comment:
this looks redundant

Reply:
Sorry the comment highlighting looks weird, the fcn `def build_lumimask(filename):` seems redundant